### PR TITLE
Added Java translation

### PR DESCRIPTION
**Pull request by igorIsyutin**
Translate 1 part of Metaprogramming. Teamwork with SergFrolov

### CODE REVIEW

File: Java/src/1-extract-metadata/1-filter-simple.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                                            name.length() <= 65 &&
+                                            name.contains("Abu") &&
+                                            name.indexOf("Lev") == 0 &&
+                                            name.endsWith("iov")
+                            )
+                    ) result.add(name);
+        }
+        return result;
+    }
+}
```
Comment (tshemsedinov) [inline Java/src/1-extract-metadata/1-filter-simple.java:46]: Add line break at the EOF

File: Java/src/1-extract-metadata/1-filter-simple.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import java.util.ArrayList;
+import java.util.List;
+
+class FilterSimple {
+    private static final String[] names = {
+            "Marcus Aurelius Antoninus Augustus",
```
Comment (tshemsedinov) [inline Java/src/1-extract-metadata/1-filter-simple.java:6]: Indentation

**Review by tshemsedinov — CHANGES_REQUESTED** [review]
Use linter

**Comment by lundibundi** [comment]
Closed due to an absence of activity. Feel free to reopen if you want to continue.

**Pull request closed without merging**